tck/src/test/java/org/eclipse/microprofile/jwt/tck/config/PublicKeyEndpoint.java

```diff
@@ -30,8 +30,11 @@
 import java.net.URL;
 import java.security.PublicKey;
 import java.security.interfaces.RSAPublicKey;
+import java.util.Arrays;
```

Comment:
Please create a dedicated `AudienceValidationEndpoint` in the `jaxrs` package

Reply:
done on 175